docs/api/rest-endpoints.md

```diff
@@ -100,3 +114,384 @@ Example
   ]
 }
 ```
+*******
+
+## Entrance
+
+### Log in
+
+`POST /api/v1/kolide/login`
+
+#### Parameters
+
+| Name                    | Type    | In    | Description                                       |
+|-------------------------|---------|-------|---------------------------------------------------|
+| username                | string  | body  | **Required**. The user's email.              |
+| password                | string  | body  | **Required**. The user's plain text password.|
+
+#### Example
+
+`POST /api/v1/kolide/login`
+
+##### Request body
+
+```
+{
+  username: "janedoe@example.com"
+  passsword: "VArCjNW7CfsxGp67"
+}
+```
+
+##### Default response
+
+`Status: 200`
+
+```
+{
+  "user": {
+    "created_at": "2020-11-13T22:57:12Z",
+    "updated_at": "2020-11-13T22:57:12Z",
+    "id": 1,
+    "username": "jane",
+    "name": "",
+    "email": "janedoe@example.com",
+    "admin": true,
+    "enabled": true,
+    "force_password_reset": false,
+    "gravatar_url": "",
+    "sso_enabled": false
+  },
+  "token": "{your token}"
+}
+
+```
+*******
+
+### Log out
+
+`POST /api/v1/kolide/logout`
+
+#### Parameters
+
+| Name                    | Type    | In     | Description                                                               |
+|-------------------------|---------|--------|---------------------------------------------------------------------------|
+| authorization           | string  | header | **Required**. The token received from the `kolide/login` response object. |
```

Comment:
I think once we have added a note about authentication with the `Authorization` header, we can remove this from the parameters of the individual endpoints (since it's going to be the same for all of them).

Reply:
Got it! Makes sense.